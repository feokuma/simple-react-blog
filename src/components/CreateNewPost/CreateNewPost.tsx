import React from "react";

function CreateNewPost() {
  return (
    <div>
      <form>
        <h1>Create New Post</h1>
        <input type="text" placeholder="Title" required />
        <br />
        <br />
        <textarea placeholder="Contents" cols={41} rows={8} required></textarea>
        <br />
        <br />
        <button>Save Post</button>
      </form>
    </div>
  );
}

export default CreateNewPost;
