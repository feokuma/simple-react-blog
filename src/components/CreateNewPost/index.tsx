import CreateNewPost from './CreateNewPost';

export default CreateNewPost;
