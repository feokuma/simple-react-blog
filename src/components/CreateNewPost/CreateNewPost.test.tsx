import React from 'react';
import renderer from 'react-test-renderer';
import CreateNewPost from './CreateNewPost';

test('Should render correctly', () => {
  const tree = renderer.create(<CreateNewPost />).toJSON();

  expect(tree).toMatchSnapshot();
});
