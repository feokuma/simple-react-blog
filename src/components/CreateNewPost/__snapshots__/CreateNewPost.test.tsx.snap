// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div>
  <form>
    <h1>
      Create New Post
    </h1>
    <input
      placeholder="Title"
      required={true}
      size={39}
      type="text"
    />
    <br />
    <br />
    <textarea
      cols={41}
      placeholder="Contents"
      required={true}
      rows={8}
    />
    <br />
    <br />
    <button
      type="submit"
    >
      Save Post
    </button>
  </form>
</div>
`;
